components/post/BestBadge.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from 'next/image';
+import styles from './BestBadge.module.scss';
+
+function BestBadge() {
+  return (
+    <>
+      <div className={styles.badge}>
+        <Image src='images/ic_medal.svg' width={16} height={16} alt='medal' />
+        <p>Best</p>
+      </div>
+    </>
```

Comment:
```suggestion
      <div className={styles.badge}>
        <Image src='images/ic_medal.svg' width={16} height={16} alt='medal' />
        <p>Best</p>
      </div>
```
특별한 이유가 없다면 빈 fragment안에 children이 하나면 지워주시면 좋습니다!